k8s/namespaces/fees-pay/dead-letter-queue-process/demo-00.yaml

```diff
@@ -5,5 +5,6 @@ metadata:
 spec:
   values:
     job:
+      image: "hmctspublic.azurecr.io/payment/jobs:prod-0a0ded6-20210716134918"   #{"$imagepolicy": "flux-system:demo-dead-letter-queue-process"}
```

Comment:
This imagepolicy is going to pull a PR image: pr-30-* - do you need to update this?

Reply:
Yeah that is what we want, working with Hannah Smith on this to get this PR correct